### DISCUSSION TITLE
Dynamic Field Select in Pods Field Value Block and wrapper HTML

### DISCUSSION THREAD

**Discussion by poirazis** [pods-framework/pods#6817] [Ideas]
### Problem to Solve

Two enhancements on the Pods Field Value Block that would make it much easier to use.

1. Dynamic Field Select 

Instead of having to remember the field name, a dropdown with a list of field you be really nice.
<img width="283" alt="Screenshot 2022-01-31 at 12 57 06 PM" src="https://user-images.githubusercontent.com/80512170/151782157-cdf43ae8-ebdb-482f-ad29-1046608f4edd.png">

2. Wrapper Element

An option to select the wrapper element of the returned value. Like <span> / <div> / <h1> etc , instead of returning the field value unwrapped.

### Proposed Solution

Described in "Problem to Solve"

**Comment by sc0ttkclark** [comment]
This would be easier except it changes depending on which Pod you select. That's the part we have to work out here and add some custom handling.

I'd still love to make this happen.

**Comment by poirazis** [comment]
To take it one step further, i would suggest an autocomplete dynamic select for the "slug" field. Again, filtered out based on the pod you selected.

Maybe a react component/control package to spawn out the react dynamic logic.
